extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/utils/UiAssetMapper.java

```diff
@@ -57,12 +63,25 @@ public UiAsset buildUiAsset(JsonObject assetJsonLd, String connectorEndpoint, St
         var creatorOrganizationName = JsonLdUtils.string(creator, Prop.Foaf.NAME);
         creatorOrganizationName = isBlank(creatorOrganizationName) ? participantId : creatorOrganizationName;
 
+        var description = JsonLdUtils.string(properties, Prop.Dcterms.DESCRIPTION);
+        String shortDescription = null;
+
+        if(description != null) {
+            var options = new MutableDataSet();
+            Parser parser = Parser.builder(options).build();
+            HtmlRenderer renderer = HtmlRenderer.builder(options).build();
+            Node document = parser.parse(description);
+            String html = renderer.render(document);
+            shortDescription = Jsoup.parse(html).text();
+        }
```

Comment:
Let's maybe reduce the function complexity. A separate Unit `MarkdownToTextConverter` with a method `extractText(String)` would make things clean. 

Also we should maybe limit the short description to 150 characters or sth similar.